contracts/MiniMeToken.sol

```diff
@@ -369,7 +369,7 @@ contract MiniMeToken is Controlled {
         uint256 curTotalSupply = totalSupply();
         require(curTotalSupply >= _amount, "No enough supply");
         uint256 previousBalanceFrom = balanceOf(_owner);
-        require(previousBalanceFrom >= _amount, "Not enough balance");
+        require(previousBalanceFrom >= _amount, "No enough balance");
```

Comment:
Actually, these are not correct grammar. They were spelled correctly before these changes.